labs/lab2/chapter2.md

```diff
@@ -43,7 +43,7 @@ $ sudo podman build -t bigimg .
 
 To run the podman container based on the image we just built use the following command:
 ```bash
-$ sudo podman run -P --name=bigapp -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb -d bigimg
+$ sudo podman run -p 80 --name=bigapp -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb -d bigimg
```

Comment:
i think we had this before and it didn't work

Reply:
I think when I ran the lab I ran it this way and it worked. If it doesn't work then it's a bug we should open.

Reply:
@scollier can you determine if this works or not?

Reply:
just tested through ch2 and, imo, the `-P` works better in the context of this lab. it assigns a random host port that the student discovers and curls later in the lab via `sudo podman port bigapp 80`.

I know in later chapters we switch to specifying the host port w/ `-p 8080`, etc.

Reply:
`-P` and `-p 80` are equivalent. They do the exact same thing: assign a random port on host to port 80 in container. One is just more explicit and it matches the text in the paragraph better. 

@tchughesiv I think you think I'm trying to change this to `-p 80:80`. That's not what I'm trying to do.

Reply:
@dustymabe w/ docker that may have been the case, but w/ podman `-p 80` appears to assign 80 on the host.

Reply:
```shell
$ sudo podman ps
CONTAINER ID   IMAGE                             COMMAND                  CREATED AT                      STATUS             PORTS                                    NAMES
2e753fa06554   docker.io/library/bigimg:latest   /bin/bash /scripts/...   2018-04-30 15:47:17 +0000 UTC   Up 7 seconds ago   0.0.0.0:80->80/udp, 0.0.0.0:80->80/tcp   bigapp

ec2-user@ip-10-50-0-199 workspace]$ sudo podman port bigapp 80
0.0.0.0:80
```

Reply:
sounds like a bug then.. :(

Reply:
ok in that case we need to leave it like it is but probably update the text in the following paragraph. I'll try to do that today.